libs/baselibc/src/tinyprintf.c

```diff
@@ -103,15 +103,15 @@ static void li2a(long num, struct param *p)
 }
```

Comment:
In general, please remove whitespace changes from patch submissions.  Once you get the commit bit, you can make those changes, but its hard to read through the patches and distinguish what is WS and what isn't -- and those WS changes are probably not that critical.